models/databaseUrl.js

```diff
@@ -1,5 +1,5 @@
-const pg = require("pg");
-const dotenv = require("dotenv");
+const pg = require('pg');
```

Comment:
I think this file `databaseUrl.js` It can be removed it doesn't have any value. because we are using sequelizer

Reply:
I think you should use es6